Deleting workspaces with resources under management (#146)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/organizations.mdx

| `data.attributes.owners-team-saml-role-id`                                | string  | (nothing) | **Optional.** **SAML only** The name of the ["owners" team](/enterprise/user-management/saml/team-membership#managing-membership-of-the-owners-team)                                                                                                                                                                                                                               |
| `data.attributes.assessments-enforced`                                    | boolean | false     | Whether or not to compel health assessments for all eligible workspaces. When true, health assessments occur on all compatible workspaces, regardless of the value of the workspace setting `assessments-enabled`. When false, health assessments only occur for workspaces that opt in by setting `assessments-enabled: true`. Not available in Terraform Enterprise.             |

> | `data.attributes.allow-force-delete-workspaces`                           | boolean | false     | Whether or not workspace admins are permitted to [delete workspaces with resources under management](/cloud-docs/users-teams-organizations/organizations#general). If false, only organization owners may delete workspaces with resources under management.                                                                                                                                                                              |

Reviewer: ```suggestion
| `data.attributes.allow-force-delete-workspaces`                           | boolean | false     | Whether workspace administrators can [delete workspaces with resources under management](/cloud-docs/users-teams-organizations/organizations#general). If false, only organization owners may delete these workspaces.                                                                                                                                                                              |
```
